index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+ <body>
+  
+<header>
+ <nav>
+  <div class="logo-placeholder"></div>
+  <div class="logo">My logo
+  </div>
+ <div class="menu-icon">
+ <img src ="img\icons8-menu-rounded-50 (1).png">
+</div>
+ 
+<div class="background-image">
+ <img src="img\header-shapes mobile.png" alt ="">
+</div>
+</nav>
+
+<div class="header-content">
+  <div class="primary-text">I’m Mariane
+    Glad to see you!</div>
+
+    <div class="primary-text2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+
+  </div>
```

Comment:
- [ ] The width of the heading should be reduced. If you are keen you can see that the width it covers is not as same as the one the paragraph covers. Kindly try as much as possible to stick to the Figma design 👍 
- [ ] Additionally, ensure that the space between the heading and the paragraph is relatively smaller as observed 😎 in the Figma design.

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/219612764-732c20b3-d3ea-4ac7-918d-4a561eac5001.png)

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/219613367-072ae202-e754-4b4d-bc51-d69c266f0aea.png)